src/options.h

```diff
@@ -113,9 +113,9 @@
 #  define MEMDEBUG
 #  define KEEP_BODIES
 #  endif
-// Memory sanitizer also needs thread-local storage
+// Memory sanitizer also needs small memory model
```

Comment:
This flag doesn't directly toggle TLS, it toggles small memory model (and PIC). The small memory model isn't supported by the SectionMemoryManager (the result being occasional random segfaults and memory corruption). When you fix the SMM, I'll stop removing the flag.